rma_sale_mrp/models/res_company.py

```diff
@@ -0,0 +1,9 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    add_component_from_sale = fields.Boolean()
```

Comment:
Can you rename it to `rma_add_component_from_sale`? so it is clear it is an rma setting

Reply:
Done @AaronHForgeFlow